src/pages/Personopplysninger/Personopplysninger.tsx

```diff
@@ -17,8 +18,136 @@ export function Personopplysninger() {
   return (
     <Card hasLargePadding hasMargin>
       <Heading level="2" size="medium" spacing>
-        Personopplysninger
+        <FormattedMessage id="personopplysninger.header" />
       </Heading>
+      <section>
+        <Heading level="3" size="small" spacing>
```

Comment:
Ved å bruke heading level 2 og heading level 3 med spacing prop på begge ender man opp med 12px luft mellom de to. Det er skissert med 1 rem (altså 16px).
Det er grunnen til at jeg har kodet styling i Forbehold siden. Nå at det er sagt, hadde det vært fint å kunne bruke Aksel standard, så kanskje bør vi høre med Cecilie.

Reply:
Løst i slack da vi følger Aksel styling